src/components/main/Staff.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ArticleList from 'components/main/ArticleList';
+import BaseComponent from 'lib/BaseComponent';
+import { parseMarkdown } from 'lib/react-utilities';
+import _ from 'lodash';
+
+export default class Staff extends BaseComponent {
+  render() {
+    const { staff } = this.props;
+    return (
+      <div key={staff.slug} className="staff">
+        <div className="staff__header">
+          <img
+            className="staff__header__staff-image"
+            alt="staff"
+            src={staff.image_url}
+          />
+          <div className="staff__header__staff-info">
+            <h1 className="staff__header__staff-info__name">{staff.name}</h1>
+            <h2 className="staff__header__staff-info__role">
+              {staff.job_title}
+            </h2>
+            <p className="staff__header__staff-info__biography">
+              {parseMarkdown(staff.biography)}
+            </p>
+          </div>
+        </div>
+        <ArticleList articles={_.toArray(staff.articles)} />
+      </div>
+    );
+  }
+}
+
+Staff.propTypes = {
+  staff: React.PropTypes.shape({
+    name: React.PropTypes.string.isRequired,
+    biography: React.PropTypes.string,
+    title: React.PropTypes.string,
+    photo: React.PropTypes.string,
+    articles: React.PropTypes.object,
+  }),
+};
```

Comment:
General comment for renames like this. It looks like you used `mv` and then `git add` and `git rm` which when combined with the changes made git's diff engine unable to tell staff and authors are the same file. 

I'm pretty sure if you use `git mv` instead, it will make PR's like this a little bit easier to review.

Reply:
Oh interesting, I wonder if that saves some extra meta data, I never knew of that command. But yeah I basically did what you said, except in effect I used `mv` and then `git add -A` which does both the add and rm.

Reply:
https://stackoverflow.com/a/1094392/5711883 ahh, I'm afraid according to this it can't be avoided (though locally I use a better diff algorithm so maybe we can do something similar with query params for PR reviews?)

Reply:
I think all `git mv` does is adds the changes to the path to the index, but not the changes to the file itself. It might actually still have the same problems here if you did the move and the added changes in the same commit, but if you do them as two separate commits - one becomes the rename - which git understands - and the other becomes the file changes. 

Not sure of the behavior if you commit both the changes and the rename in the same commit, but either way `git mv` is not much more effort than just `mv` 😄

Reply:
Yeah I think it may even have the same problems, because I think I did do the renames in different commits, and I just tried the query params and couldn't get it to work

Reply:
Yeah Github just sucks lol, check this out https://github.com/thegazelle-ad/gazelle-server/pull/349/commits/2b2d37f0e271e1000d49e9f6762ab89e90974759

Reply:
For next time I can try to tell you guys the ranges from which actual edits were so you can check it in chunks on each side of the file rename

Reply:
Yeah, it's not a big deal since it's pretty easy to understand as long as I am doing an actual careful scan. It's just a pain that I thought `git mv` might fix, but I guess not haha.